Emby.Server.Implementations/ApplicationHost.cs

```diff
@@ -1167,7 +1167,7 @@ private IEnumerable<Type> GetTypes(IEnumerable<Assembly> assemblies)
                 {
                     exportedTypes = ass.GetExportedTypes();
                 }
-                catch (TypeLoadException ex)
+                catch (FileNotFoundException ex)
```

Comment:
This is used for more than just loading plugins, are we sure we want to swallow exceptions for everything here? It might be better to do this only for plugins and allow the core things to continue fail

Reply:
This PR just fixes the originally intended behavior. And changing it would require a rework of how plugins are loaded (if we want to do it properly)

Reply:
Fair enough, LGTM (let's get this merged)